server/middlewares.js

```diff
@@ -8,10 +8,10 @@ import filterCollection from '../frontend/src/lib/filter_collection';
 /**
  * Fetch users by slug
  */
-const fetchUsers = (options) => {
+const fetchActiveUsers = (options) => {
   return (req, res, next) => {
     api
-      .get(`/groups/${req.params.slug}/users`, options)
+      .get(`/groups/${req.params.slug}/active_users`, options)
```

Comment:
I'd suggest to add an option to the `users` endpoint instead of introducing new one